Description: Upstream changes introduced in version 4.8.2-0ubuntu2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 unity (4.8.2-0ubuntu2) oneiric; urgency=low
 .
   * debian/control: really update the nux requirement
 .
 The person named in the Author field signed this changelog entry.
Author: Sebastien Bacher <seb128@ubuntu.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- unity-4.8.2.orig/services/CMakeLists.txt
+++ unity-4.8.2/services/CMakeLists.txt
@@ -39,7 +39,7 @@ set(CFLAGS
     "-DGETTEXT_PACKAGE=\"unity\""
     "-DINDICATORDIR=\"${_indicatordir}\""
     "-DINDICATORICONDIR=\"${_iconsdir}\""
-    "-Werror -Wall"
+    "-Wall"
     )
 add_definitions(${CFLAGS})
 
--- unity-4.8.2.orig/plugins/unityshell/src/DashView.cpp
+++ unity-4.8.2/plugins/unityshell/src/DashView.cpp
@@ -165,6 +165,11 @@ nux::Geometry DashView::GetBestFitGeomet
   height += tile_height * 4.75;
   height += lens_bar_->GetGeometry().height;
 
+  if (for_geo.width > 800 && for_geo.height > 550)
+  {
+    width = MIN(width, for_geo.width-66);
+    height = MIN(height, for_geo.height-24);
+  }
   return nux::Geometry(0, 0, width, height);
 }
 
--- unity-4.8.2.orig/plugins/unityshell/src/BackgroundEffectHelper.cpp
+++ unity-4.8.2/plugins/unityshell/src/BackgroundEffectHelper.cpp
@@ -38,6 +38,7 @@ BackgroundEffectHelper::BackgroundEffect
 
 BackgroundEffectHelper::~BackgroundEffectHelper()
 {
+  noise_texture_->UnReference();
   Unregister(this);
 }
 
